packages/js/src/workouts/components/ConfigurationWorkout.js

```diff
@@ -405,9 +404,9 @@ export function ConfigurationWorkout( { toggleStep, finishSteps, reviseStep, tog
 
 	/* eslint-disable max-len */
 	return (
-		<div className="card">
-			<h2>{ __( "Configuration", "wordpress-seo" ) }</h2>
-			<h3>{
+		<div id="yoast-configuration-workout-card" className="card">
+			<h2 id="yoast-configuration-workout">{ __( "Configuration", "wordpress-seo" ) }</h2>
+			<h3 id="yoast-configuration-workout-title">{
```

Comment:
Is any of this needed in the Premium workouts as well?

Reply:
Yes it is, but for now I just added it to the configuration workout since Dario is working on that right now. We can add them to the Premium workouts in another issue.